backend/handler/handler.go

```diff
@@ -32,8 +36,9 @@ type InitializeResponse struct {
 }
 
 type registerRequest struct {
-	Name     string `json:"name"`
-	Password string `json:"password"`
+	// name and password are set be aplhanumeric only
+	Name     string `json:"name" validate:"required,min=4,max=20,alphanum"`
+	Password string `json:"password" validate:"required,min=6,max=20,alphanum"`
```

Comment:
for hackathon this is acceptable, but not in real life app

Reply:
I see. I will go ahead and revise it.

Reply:
we can push this once you make this change

Reply:
Have some other ideas that I want to change altogether. Will do it after I finish.